sklearn/preprocessing/_encoders.py

```diff
@@ -796,6 +815,10 @@ def inverse_transform(self, X):
         """
         Convert the data back to the original representation.
 
+        If there are categories that are missing, then
+        both `None` and `np.nan` are inverse transformed as `np.nan`, i.e.
+        `inverse_transform(transform([None])) == np.nan`.
```

Comment:
It seems that this snippet doesn't illustrate the above textual description:

`transform([None])` is `[nan]` so all this is saying is that `inverse_transform([nan]) == [nan]`, as expected

Instead the text above suggests describes the fact that `inverse_transform([None]) == [nan]` but:

- this doesn't seem to be tested (I might have missed something)
- I find that quite unexpected since `transform()` is unable to produce None as output. So I would expect `inverse_transform` to fail if we give it None, or any other unexpected value (like a floating value)

Reply:
I adjusted the wording here:

> If a categorical feature uses `None` *and* `np.nan` as a missing value indicator, the inverse_transform will map `np.nan` to `np.nan`.

This behavior is tested in `test_ordinal_encoder_passthrough_missing_values_object`. (I added a comment there to explicitly say this.)